prime-angular-frontend/src/app/core/services/error-handler.service.ts

```diff
@@ -29,16 +32,7 @@ export class ErrorHandlerService implements ErrorHandler {
       }
     } else {
       // Client error has occurred (Angular Error, ReferenceError...)
+      dialogLogger.log(error);
     }
-
-    const message = (error.message)
-      ? error.message
-      : error.toString();
-    const url = router.url;
-
-    logger.error(message, { url });
-
-    // IMPORTANT: Rethrow the error, otherwise it gets swallowed
-    throw error;
```

Comment:
I'm pretty sure we need to keep this in there right?

Reply:
yep deleted by accident.